weed/config/s3_config.go

```diff
@@ -0,0 +1,16 @@
+package config
```

Comment:
please find a good directory

Reply:
That's right, I originally wanted to put this in the s3api directory, but there is a circular dependency, so I temporarily put it in the outer layer;
package github.com/chrislusf/seaweedfs/weed
  imports github.com/chrislusf/seaweedfs/weed/command
  imports github.com/chrislusf/seaweedfs/weed/iamapi
  imports github.com/chrislusf/seaweedfs/weed/s3api
  imports github.com/chrislusf/seaweedfs/weed/server
  imports github.com/chrislusf/seaweedfs/weed/shell
  imports github.com/chrislusf/seaweedfs/weed/s3api: import cycle not allowed

Reply:
any suggestion?